controllers/capabilities/tenant_controller.go

```diff
@@ -122,7 +122,7 @@ func (r *TenantReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 					return ctrl.Result{}, err
 				}
 			} else {
-				reqLogger.Info("Removing tenant CR - tenant is already scheduled for deletion %v", existingTenant.Signup.Account.ID)
+				reqLogger.Info("Error", "Removing tenant CR - tenant is already scheduled for deletion %v", existingTenant.Signup.Account.ID)
```

Comment:
Good catch!

In logr, format strings are not allowed [link](https://github.com/go-logr/logr#why-not-allow-format-strings-too)

It should be:
```go
reqLogger.Info("Removing tenant CR - tenant is already scheduled for deletion", "tenantID", existingTenant.Signup.Account.ID)
```

Reply:
Ah, okay. Made the change.